ruby_programming/computer_science/project_data_structures_algorithms.md

```diff
@@ -1,11 +1,11 @@
 ### Project 1: Binary Search Trees
 
-You learned about [binary search trees](http://en.wikipedia.org/wiki/Binary_search_tree) -- where you take a group of data items and turn them into a tree full of nodes where each left node is "lower" than each right node.  The tree starts with the "root node" and any node with no children is called a "leaf node".
+You have learned about [binary search trees](http://en.wikipedia.org/wiki/Binary_search_tree) -- where you take a group of data items and turn them into a tree full of nodes where each left node is "lower" than each right node.  The tree starts with the "root node" and any node with no children is called a "leaf node". You have also learned about tree traversal algorithms like breadth-first and depth-first.
 
-You also learned about tree traversal algorithms like breadth-first and depth-first which we'll attempt to implement here.
+Now, let's take a look at balanced binary search trees. Read [this article](https://www.geeksforgeeks.org/sorted-array-to-balanced-bst/) and watch [this video](https://youtu.be/VCTP81Ij-EM) to understand the basic algorithm used to build a balanced BST. Although these two resources do not use Ruby, you should understand it enough to develop your own pseudocode.
```

Comment:
Grammatical, but the acronym should be defined after the fully written words: `Now, let's take a look at balanced binary search trees (BST)`.

After that, it can be used freely.